samples/servicebus-queue/readme.md

```diff
@@ -153,13 +156,30 @@ kubectl get --raw "/apis/external.metrics.k8s.io/v1beta1" | jq .
 }
 ```
 
+### Configure Metric Adapter with metrics
+The metric adapter deploys a CRD called ExternalMetric which you can use to configure metrics.  To deploy these metric we need to update the Service Bus namespace in the configuration then deploy it:
+
+```
+sed -i 's|sb-external-ns|'${SERVICEBUS_NS}'|g' deploy/externalmetric.yaml
+kubectl apply -f deploy/externalmetric.yaml
+```
+
+You can list of the configured external metrics via:
+
+```
+kubectl get aem #shortcut for externalmetric
+
+```
+
 ### Deploy the HPA
 Deploy the HPA:
 
 ```
 kubectl apply -f deploy/hpa.yaml
 ```
 
+> note: the `external.metricName` defined on the HPA must match the `metadata.name` on the ExternalMetric HPA, in this case `queuemessages`.
```

Comment:
Shouldn't this be something like the following?
> note: the `external.metricName` defined on the HPA must match the `metadata.name` on the ExternalMetric declaration, in this case `queuemessages`.

Hence the "ExternalMetric HPA" -> "ExternalMetric declaration"